config/development.json

```diff
@@ -59,6 +59,7 @@
 		"help/courses": true,
 		"jetpack/api-cache": true,
 		"jetpack/connect-redirect-pressable-credential-approval": true,
+		"jetpack/connect/install": true,
```

Comment:
Wondering if we should make this flag `jetpack/connect/remote-install` to differentiate it from the existing manual install instructions and match up with the [new state](https://github.com/Automattic/wp-calypso/tree/master/client/state/jetpack-remote-install)?